src/components/Integrations/Shopify/Shopify.js

```diff
@@ -90,16 +93,34 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getSubscribersAmountFromAPI = async (listId, apikey) => {
+    const resultAPI = await dopplerAPIClient.getListData(listId, apikey);
+    if (resultAPI.success) {
+      return resultAPI.value.amountSubscribers;
+    }
+    return 0;
```

Comment:
I think that showing zero when there is an error is not a good idea.

Reply:
It would be an api error and not a shopify error, perhaps in this case show shopify customers? or log the error...